boards/iotlab-a8-m3/doc.txt

```diff
@@ -0,0 +1,4 @@
+/**
+ * @ingroup     boards_iotlab-m3
```

Comment:
There is a problem here, the group should be `boards_iotlaba-a8-m3`. This is because the initial group of the A8 was invalid.